mod-source-record-manager-server/src/main/java/org/folio/dao/JobExecutionDaoImpl.java

```diff
@@ -205,6 +206,19 @@ public Future<JobExecution> updateBlocking(String jobExecutionId, JobExecutionMu
     return promise.future();
   }
 
+  @Override
+  public Future<Boolean> deleteJobExecutionsByIds(List<String> ids, String tenantId) {
+    Promise<RowSet<Row>> promise = Promise.promise();
+    try {
+      String query = format(DELETE_BY_IDS_SQL, convertToPsqlStandard(tenantId), TABLE_NAME, ids.stream().collect(Collectors.joining("','")));
+      pgClientFactory.createInstance(tenantId).execute(query, promise);
+    } catch (Exception e) {
+      LOGGER.error("Error getting jobExecution by id", e);
```

Comment:
Please correct message to: LOGGER.error("Error deleting job executions by ids: {}", ids, e);

Reply:
@SerhiiNosko - PR updated as per comments, Thanks 🙂